Examples/Io/Performance/ActsExamples/Io/Performance/VertexPerformanceWriter.cpp

```diff
@@ -673,13 +673,11 @@ ActsExamples::ProcessCode ActsExamples::VertexPerformanceWriter::writeT(
               }
 
               // Save track parameters after the vertex fit
-              const auto paramsAtVtxFitted = propagateToVtx(trk.fittedParams);
-              if (paramsAtVtxFitted != std::nullopt) {
+              if (trk.fittedMomentum != std::nullopt) {
                 Acts::ActsVector<3> recoMomFitted =
-                    paramsAtVtxFitted->parameters().segment(Acts::eBoundPhi, 3);
+                    trk.fittedMomentum.value().momentum;
```

Comment:
```suggestion
                    trk.fittedMomentum->momentum;
```

`value()` will actually check if there is a value and throw and exception otherwise. since we know that the optional has a value we can directly access the memory and skip the condition